server/server.go

```diff
@@ -703,12 +711,19 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 		userConfig.SilenceNoProjects,
 	)
 
+	importCommandRunner := events.NewImportCommandRunner(
+		pullUpdater,
+		projectCommandBuilder,
+		instrumentedProjectCmdRunner,
```

Comment:
I've noticed that these structs don't have the instrumented functions, I believe this would be a nice thing to have, and fairly easy too, this would suffice:
- Add `Import(ctx command.ProjectContext) command.ProjectResult` implementation to `server/events/instrumented_project_command_runner.go` for `project` metrics
- Add `BuildImportCommands(ctx *command.Context, comment *CommentCommand) ([]command.ProjectContext, error)` implementation to `server/events/instrumented_project_command_builder.go` for `builder` metrics

Reply:
added thanks! https://github.com/runatlantis/atlantis/pull/2783/commits/bed8ec0d7edac6abc6313eaface5ce493ec56369